custom/icds/messaging/custom_recipients.py

```diff
@@ -1,37 +1,73 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 from corehq.apps.locations.models import SQLLocation
+from corehq.form_processor.models import CommCareCaseIndexSQL
 from custom.icds.case_relationships import (
     mother_person_case_from_ccs_record_case,
     mother_person_case_from_child_health_case,
     mother_person_case_from_child_person_case,
 )
 from custom.icds.const import SUPERVISOR_LOCATION_TYPE_CODE
 from custom.icds.exceptions import CaseRelationshipError
+from datetime import datetime
 from dimagi.utils.logging import notify_exception
 
 
+def skip_notifying_missing_mother_person_case(e):
+    # https://manage.dimagi.com/default.asp?271995
+    # It's expected that some child person cases will not have a mother person case,
+    # so we don't notify when that's the lookup that fails.
+
+    return (
+        e.child_case_type == 'person' and
+        e.identifier == 'mother' and
+        e.relationship == CommCareCaseIndexSQL.CHILD and
+        e.num_related_found == 0
+    )
+
+
+def skip_notifying_missing_ccs_record_parent(e):
+    # https://manage.dimagi.com/default.asp?277600
+    # This is an open issue so it probably doesn't make sense to keep notifying
+    # these unless it gets resolved. Going to make these start notifying at a
+    # later date so this can be revisited.
+
+    return (
+        datetime.utcnow() < datetime(2018, 8, 1) and
```

Comment:
this one will start notifying again in august as a reminder to check in on the ticket. once the issue is resolved we should allow notifying again